code/modules/reagents/chemistry/reagents/impure_reagents/impure_medicine_reagents.dm

```diff
@@ -591,17 +591,15 @@ Basically, we fill the time between now and 2s from now with hands based off the
 	if(!carbon.dna)
 		return
 	var/list/speech_options = list(SWEDISH, UNINTELLIGIBLE, STONER, MEDIEVAL, WACKY, NERVOUS, MUT_MUTE)
-	while(speech_options || !speech_option)
-		var/potential_option = pick(speech_options)
+	while(length(speech_options) || !speech_option)
+		var/potential_option = pick_n_take(speech_options)
+		if(!potential_option)
```

Comment:
I don't see how this check can ever be true here and below, these are static list no nulls here.

Reply:
Ahh I'll just rewrite it to not use a while loop